src/DotUtils.MsBuild.SensitiveDataDetector/SensitiveDataDetectorFactory.cs

```diff
@@ -6,37 +6,55 @@
 namespace Microsoft.Build.SensitiveDataDetector;
 
 [Flags]
-public enum SensitiveDataKind
+public enum SensitiveDataKind : long
```

Comment:
Btw. this won't scale up pretty soon.
We can then make this a normal enum and the requesting of the types to scan should not be possible to be that picky (probably just by categories of confidence, or maybe even not that - we should just allways scan everything. Only username feels as something to be possibly legit to opt-out)

Reply:
I reverted it to the initial state